From 51dfcfebf54b114f9bfaa2b0d6554d985e27c3c9 Mon Sep 17 00:00:00 2001
From: Robby Cai <robby.cai@nxp.com>
Date: Fri, 8 Jan 2016 18:51:07 +0800
Subject: [PATCH 0830/5242] MLK-12190 ARM: dts: imx7d-sdb-revb: change pad
 setting to remove stripe

commit  8e39ebf6915c7de77a277b331c521dc59e770af6 from
https://source.codeaurora.org/external/imx/linux-imx.git

Change GPIO pad setting to control EPDC/ENET signals on REV B board from 0x59
to default one 0x14 in order to remove stripe when do EPDC unit test.

Signed-off-by: Robby Cai <robby.cai@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7d-sdb.dts |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx7d-sdb.dts b/arch/arm/boot/dts/imx7d-sdb.dts
index bb928aa..bfd6d4a 100644
--- a/arch/arm/boot/dts/imx7d-sdb.dts
+++ b/arch/arm/boot/dts/imx7d-sdb.dts
@@ -840,7 +840,7 @@
 
 		pinctrl_enet2_epdc0_en: enet2_epdc0_grp {
 			fsl,pins = <
-				MX7D_PAD_LPSR_GPIO1_IO04__GPIO1_IO4	0x59
+				MX7D_PAD_LPSR_GPIO1_IO04__GPIO1_IO4          0x80000000
 			>;
 		};
 
-- 
1.7.9.5

